content/03.when-to-use-dl.md

```diff
@@ -1,51 +1,45 @@
+In the future, projects such as these are likely to make deep learning increasingly accessible to a much wider swatch of researchers.
 
-There are some types of problems in which using deep learning is strongly indicated over machine learning.
-Assuming a sufficient quantity of quality data is available, applications such as computer vision and natural language processing are likely to benefit from deep learning.
-In fact, these areas were the first to see significant breakthroughs through the application of deep learning [@doi:10.1145/3065386] during the recent deep learning revolution."
-For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436].
-This problem was historically difficult but, by combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning achieve 90% accuracy in several species.
-Other areas include generative models, in which new samples are able to be created based on the training data, and reinforcement learning, in which agents are trained to interact with their environments.
-In general, before using deep learning, investigate whether similar problems (including analogous ones in other domains) have been solved successfully using deep learning.
+Despite these limitations, deep learning is strongly indicated over traditional machine learning for certain research questions and problems.
+In general, these include problems that feature hidden patterns across the data, complex relationships, and interrelated variables.
+Problems in computer vision and natural language processing often exhibit exhibit these very features, which helps to explain why these areas were some of the first to experience significant breakthroughs during the recent deep learning revolution [@doi:10.1145/3065386]. 
+As long as large amounts of accurate and labeled data are available, applications to areas of biology with related data characteristics, such as genetic medicine [@doi:10.3389/fpsyt.2018.00290], radiology [@doi:10.1007/s11604-018-0726-3], microscopy [@doi:10.1364/OPTICA.4.001437], and pharmacovigilance [@doi:10.1093/jamia/ocw180], are similarly likely to benefit from deep learning techniques.
+For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436] despite this problem being very difficult historicallly. By combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning to achieve 90% accuracy in several species.
```

Comment:
```suggestion
For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436] despite this problem being very difficult historicallly.
By combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning to achieve 90% accuracy in several species.
```
Fixed one sentence per line but not sure it makes sense to use the cross-referencing tips.

Reply:
Going to delete the cross references in a future PR so it's ok to leave them for now